### ISSUE TITLE
Wrong permissions

### ISSUE THREAD

**Issue by m1ga** [dbankier/tisdk#3]
It looks like the permissions are wrong when installing through tisdk:

```
tisdk install 4.1.0.GA
```

then trying to compile leads to:

```
events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: spawn EACCES
    at errnoException (child_process.js:1011:11)
    at ChildProcess._handle.onexit (child_process.js:802:34)
```

changing the whole 4.1.0.GA folder with "chmod +x" (for testing) and everything works fine.

**Comment by dbankier** [comment]
Yep. Executable permission are lost (need to fix that).
A global `+x` is not a good idea.

So far these are the two files that need `+x` permissions:
- `{sdk}/node_modules/ioslib/lib/ios-sim`
- `{sdk}/android/titanium_prep.macos`

**Comment by dbankier** [comment]
new version pushed that fixes it 1.2.0
